database/buildSchema/37_application_list_view.sql

```diff
@@ -15,13 +19,15 @@ CREATE OR REPLACE VIEW "public".application_list AS
 	-- 	template_questions_count(app),
 	-- 	assigned_questions_count(app, stage_status.stage_id, 1),
 		assigned_questions_count(app, stage_status.stage_id, 1) >= template_questions_count(app)
-		AS is_fully_assigned_level_1
+		AS is_fully_assigned_level_1,
+		review_status_list.*
 	FROM application app LEFT JOIN
 	template ON app.template_id = template.id LEFT JOIN
 	"user" ON user_id = "user".id LEFT JOIN
 	application_stage_status_latest AS stage_status
 	on app.id = stage_status.application_id
-	LEFT JOIN organisation org ON app.org_id = org.id;
+	LEFT JOIN organisation org ON app.org_id = org.id
+	LEFT JOIN review_status_list on review_status_list.application_id = app.id
```

Comment:
I wonder if this View should also return one set of applications that are not joined to a reviewer_id (i.e. all the review columns would be NULL). That way it would still be possible to query without a review_id (or reviewer_id = NULL) and get the full set of applications for a list that doesn't need to care about reviewer id.

Cos what happens to the application list for applicants. If they have a bunch of applications in progress, they'd want to be able to see them all without caring who the reviewer is, right?

You could maybe do a UNION of this query with the original query, perhaps?